src/main/java/bowling/view/ViewResult.java

```diff
@@ -0,0 +1,32 @@
+package bowling.view;
+
+import java.util.Optional;
+
+import bowling.domain.result.Result;
+import bowling.domain.score.Score;
+import bowling.domain.score.Scores;
+
+public class ViewResult {
+
+	private final Result result;
+
+	ViewResult(Result result) {
+		this.result = result;
+	}
+
+	public static String printScores(Scores scores) {
+		Optional<Score> first = Optional.ofNullable(scores.getFirst()).orElseThrow(() ->
+			new IllegalArgumentException("not found"));
+		Optional<Score> second = Optional.ofNullable(scores.getSecond()).orElseThrow(() ->
+			new IllegalArgumentException("not found"));
+
+		if (! first.isPresent()) {
+			return "      ";
+		}
+		if (! (first.map(Score::getScore).get() == 10) && ! second.isPresent()) {
+			return String.format("   %s    ", first.get().getScore());
+		}
+		Result result = scores.checkResult();
+		return result.renderString(scores);
+	}
```

Comment:
이 부분 코드는 조금 이해가 되지 않네요

orElseThrow를 한 시점에 이미 없으면 예외가 발생하는데 다시 Optional로 포장할 필요가 없을 것 같네요

제가 이해한게 맞다면 아래와 같이 정의되면 될 것 같습니다.

```suggestion
	public static String printScores(Scores scores) {
		Score first = scores.getFirst().orElseThrow(() ->
			new IllegalArgumentException("not found"));

		if (first.getScore() == 10 && scores.isSecondScoreNull()) {
			return String.format("   %s    ", first.getScore());
		}

		Result result = scores.checkResult();
		return result.renderString(scores);
	}
```